.gitignore

```diff
@@ -2,4 +2,5 @@
 source/.DS_Store
 images/.DS_Store
 updateZip.bat
-den.bat
\ No newline at end of file
+den.bat
+/dist
```

Comment:
I'm not sure where the 'dist' directory comes from. Is that something you've created?